### ISSUE TITLE
Retrieve full user model from author of a video

### ISSUE THREAD

**Issue by nTraum** [kylejginavan/youtube_it#187]
Hi,

I'm currently trying to figure out how to fetch profile information from the uploader of a video.

Assuming I'm fetching the video info with `Client#video_by 'foo'`, I have an instance of `Video` with only limited author information (`Video#author`).
Is there any way to directly access the `User` model of the uploader?

I couldn't find an attribute for the username (which I could use to query the user info with `Client#profile 'bar'`) either. Maybe I'm missing the point.

tl;dr: Want to fetch profile information from the uploader of a specified video.

Thanks in advance.
